wazp/callbacks.py

```diff
@@ -3,16 +3,21 @@
 
 # import pdb
 import re
-from typing import Any
+from typing import Any, Optional
 
 import dash
 import dash_bootstrap_components as dbc
+import plotly.express as px
+import plotly.graph_objs as go
```

Comment:
Pickiest of the picky comments: it seems like these abbreviations are "common". As-in dash use them in their own docs and example scripts.

But you only actually use them in two or three active lines each (the docstring should probably always have the full object). Perhaps "explicit is better than implicit"?

Reply:
The Zen has been evoked, and I must comply 😉 
Joke aside, thanks for his comment, I wasn't sure how to handle this in docstrings.

Reply:
🧘‍♀️